packages/connect-web/src/lib.dom.streams.ts

```diff
@@ -0,0 +1,32 @@
+// Copyright 2021-2022 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// The name for the ReadableStreamDefaultReadResult type changed in TypeScript 4.8.2 to ReadableStreamReadResult.  This
+// makes supporting multiple versions of TypeScript difficult if we use the ambient type declaration.  Instead, this
+// type is meant to mimic the type provided by TypeScript, just defined here so that we can have better control with
+// supporting various versions.  The suffix 'Like' has been added to the type name to differentiate it from the actual
+// type provided by TypeScript libs.
```

Comment:
Comment is on point :+1: 

Couple of comment style nits though:

Can you leave an empty line before the interface declaration though, so that the comment isn't attached to it?
Alternative, you could attach it to the exported element. In that case I'd recommend to move the exported type to the top, in front of the two interfaces.

Can you wrap the lines at 80 columns?

Reply:
Yep.  Done.  I added the comment to the exported element.